site2/docs/io-kafka-source.md

```diff
@@ -95,7 +123,7 @@ Here is an example of using the Kafka source connecter with the configuration fi
 5. Pull a Pulsar image and start Pulsar standalone.
    
    ```bash
-   $ docker pull apachepulsar/pulsar:2.4.0
+   $ docker pull apachepulsar/pulsar:2.8.0
```

Comment:
If the version should be updated with release, maybe we can use variable
`{{pulsar:version}}`

Reply:
thanks I didn't find the variable